class-wc-calypso-bridge.php

```diff
@@ -34,6 +34,13 @@ class WC_Calypso_Bridge {
 	 */
 	protected static $instance = null;
 
+	/**
+	 * Logger instance.
+	 *
+	 * @var WC_Logger $logger
+	 */
+	protected static $logger = null;
```

Comment:
I would suggest using the native `wc_get_logger` function that includes some kind of static management and doesn't instantiate the logger each time. You can see an example in gift cards here: https://github.com/woocommerce/woocommerce-gift-cards/blob/master/woocommerce-gift-cards.php#L377-L387

Reply:
Thank you Chris - I applied your comments!